js/providers/Model.js

```diff
@@ -111,6 +111,7 @@ define(
 				this.includedSourcecodes = ko.observableArray();
 				this.cohortDefinitions = ko.observableArray();
 				this.currentCohortDefinition = ko.observable();
+				this.cohortDefinitionSubscriptions = [];
```

Comment:
Let's move this field out of Model.js (we're trying to get rid of Model.js completely since it has a heap of different functions that should be refactored out to either services or the components these behaviors belong to).

In this case, this subscription list is being populated by the `conceptset-warning.js` , so let's move this array over to conceptset-warnings.js, and push/dispose these subscriptions when that component disposes.